### DISCUSSION TITLE
Project name etymology

### DISCUSSION THREAD

**Discussion by 3TUSK** [Sinytra/Connector#961] [Q&A] [answered]
I can't find a good spot to ask this question, so I am putting this one at the best place I believed. 

How was the name "Sinytra" decided? Are there special meaning behind this name? Is this a made-up word, or borrowed from another language?

**Comment by Su5eD** [comment, marked as answer]
Yes

**Comment by FinnT730** [comment]
And we never found the answer again
![image](https://github.com/Sinytra/Connector/assets/7442674/785430be-332d-4a50-9a2a-fa1829ad7972)

**Reply by 3TUSK** [reply]
https://discord.com/channels/313125603924639766/313125603924639766/1227693763502276680

> 3TUSK: I have [been] wondering this for a while  
> 3TUSK: what does “sinytra” mean?  
> Su5eD: hmm I think curle might know  
> 3TUSK: uh oh  
> 3TUSK: is curle here  
> 3TUSK: likely  
> Su5eD: curle are you here? we need your wisdom  
> Curle: <Replying Su5eD> i am indeed  
> Curle: <Replying 3TUSK> it’s a Frank Sinatra reference  
> Curle: and if it isn’t, it’s probably somethin’ stupid  
> Curle: https://open.spotify.com/track/4feXcsElKIVsGwkbnTHAfV

**Reply by FinnT730** [reply]
Yeah, I was kind of there XD
But it is still not an answer 🙃
